integration/convert_test.go

```diff
@@ -229,3 +232,72 @@ func TestConvertAndPush(t *testing.T) {
 		})
 	}
 }
+
+func TestInvalidConversion(t *testing.T) {
+	registryConfig := newRegistryConfig()
+	sh, done := newShellWithRegistry(t, registryConfig)
+	defer done()
+
+	tests := []struct {
+		name     string
+		repo     string
+		modifier func(t *testing.T, sh *shell.Shell, img imageInfo)
+	}{
+		{
+			name: "deleting manifest invalidates image",
+			repo: "manifest",
+			modifier: func(t *testing.T, sh *shell.Shell, img imageInfo) {
+				digest, err := getManifestDigest(sh, img.ref, platforms.DefaultSpec())
+				if err != nil {
+					t.Fatalf("failed to get manifest digest: %v", err)
+				}
+				sh.X("ctr", "content", "delete", digest)
+			},
+		},
+		{
+			name: "deleting soci index invalidates image",
+			repo: "sociindex",
+			modifier: func(t *testing.T, sh *shell.Shell, img imageInfo) {
+				index, err := getImageIndex(sh, img.ref)
+				if err != nil {
+					t.Fatalf("failed to get image index: %v", err)
+				}
+				idx := slices.IndexFunc(index.Manifests, func(desc ocispec.Descriptor) bool {
+					return desc.ArtifactType == soci.SociIndexArtifactTypeV2
+				})
+				if idx == -1 {
+					t.Fatalf("no soci index found")
+				}
+				sh.X("ctr", "content", "delete", index.Manifests[idx].Digest.String())
+			},
+		},
+	}
+
+	for _, imageName := range convertImages {
+		t.Run(imageName, func(t *testing.T) {
+			rebootContainerd(t, sh, "", "")
```

Comment:
Just double checking, this reboot is clearing the cache and needs to occur for each image so no one-time setup is possible?

Reply:
It also starts containerd for the first time. I think we could technically move this out of the loop, but clearing and restarting containerd/soci is not the expensive part of this test.

Reply:
Okay yeah thanks was just curious if it was intentional or required. LGTM.